README.md

```diff
@@ -83,6 +83,22 @@ For `zsh`, you must then add the following line in your `~/.zshrc` before
 fpath+=~/.zfunc
 ```
 
+#### Choosing where to install
+
+`rustup` allows you to customise your installation by setting the environment
+variables `CARGO_HOME` and `RUSTUP_HOME` while running the `rustup-init`
```

Comment:
I think saying "while" might give the wrong impression, despite the clarification below. Maybe say "before running" instead?